docs/runtime-compliance-testing.md

```diff
@@ -6,4 +6,375 @@ In order to be tested for [compliance][], runtimes MUST support at least one of
 
 * Version 1.0.0 of the [OCI Runtime Command Line Interface](command-line-interface.md).
 
-[compliance]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc4/spec.md#notational-conventions
+## Running the runtime validation suite
+
+The runtime validation suite uses [node-tap][], which is packaged for some distributions (for example, it is in [Debian's `node-tap` package][debian-node-tap]).
+If your distribution does not package node-tap, you can install [npm][] (for example, from [Gentoo's `nodejs` package][gentoo-nodejs]) and use it:
+
+```console
+$ npm install tap
+```
+### From a release
+
+Check if your release has pre-compiled tests on the [release page][releases] page.
+
+```
+$ tar xf runtime-tools-v0.6.0.tar.gz
+$ RUNTIME=runc tap ./runtime-tools-v0.6.0/validation/*.t
```

Comment:
I think we should be consistent about whether or not we recommend using `sudo` for the tests.  In the “From source” section below, we recommend using `sudo`, so I think we want to also recommend it here.